### PR TITLE
2146: fixed inability to scroll after submitting

### LINKED ISSUE #2146: Can't scroll after submitting
**Describe the bug**
Scrolling is disabled when going back to dashboard after submitting

**Steps To Reproduce**
* Go through a submission
* After clicking accept, click the button on the thank you page
* After you are redirected back, the window doesn't scroll. If you don't have that many submissions, start a new submission.

**Expected behavior**
The window should scroll

### PR DESCRIPTION
**Pull request by erezmus**
#### Background

Fixes bug that stops users from scrolling once they have submitted

#### Any relevant tickets

Closes #2146

### CODE REVIEW

**Comment by diversemix** [comment]
How's it been tested? Did you want to complete that section in the template? Could we put a test in somewhere?

**Comment by diversemix** [comment]
Can someone educate me as to why we fettle with the scrolling behaviour in the first place?

**Comment by erezmus** [comment]
@diversemix yep, was gonna add a test for the modal overlay, just wanted to see if it passes ci first (should have put WIP in title as well i guess)

The reason we did this way if i recall correctly is because of a previous bug (#1980) where on some browsers one couldn't scroll the list of editors.

File: packages/component-elife-ui/client/molecules/ModalOverlay.test.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    const { baseElement } = render(
+      <ModalOverlay open={false} transparentBackground>
+        Modal Content
+      </ModalOverlay>,
+      { wrapper: setupProvider },
+    )
+
+    expect(baseElement).not.toHaveStyle(`overflow: hidden`)
+
+    cleanup()
```
Comment (Cooryd) [inline packages/component-elife-ui/client/molecules/ModalOverlay.test.js]: Could be put in an `afterEach()` ?

**Review by diversemix — APPROVED** [review]

File: packages/component-elife-ui/client/molecules/ModalOverlay.test.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { render, cleanup } from '@testing-library/react'
+import theme from '@elifesciences/elife-theme'
+import { ThemeProvider } from 'styled-components'
+import { MockedProvider } from 'react-apollo/test-utils'
+import 'jest-dom/extend-expect'
+import ModalOverlay from './ModalOverlay'
+
+theme.transitionDuration = '0.2s'
```
Comment (Cooryd) [inline packages/component-elife-ui/client/molecules/ModalOverlay.test.js]: Why does this need to be over-ridden here?

**Review by will-byrne — APPROVED** [review]

File: packages/component-elife-ui/client/molecules/ModalOverlay.test.js (hunk 1 above)
Comment (erezmus) [reply packages/component-elife-ui/client/molecules/ModalOverlay.test.js]: Done and also refactored component render inside factory function

**Review by Cooryd — DISMISSED** [review]

File: packages/component-elife-ui/client/molecules/ModalOverlay.test.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    </ModalOverlay>,
+    { wrapper: setupProvider },
+  )
+
+describe('ModalOverlay', () => {
+  afterEach(() => cleanup())
+
+  it('should disable scrolling on body element when open', () => {
+    const { baseElement, rerender } = factory({ open: true })
+    rerender(
```
Comment (Cooryd) [inline packages/component-elife-ui/client/molecules/ModalOverlay.test.js:28]: do we need this rerender?

**Pull request merged**